src/components/Divider.tsx

```diff
@@ -44,7 +39,8 @@ const Divider = ({ path }: { path: Path }) => {
         maxWidth: '100%',
         padding: '10px 4px 16px',
         position: 'relative',
-        width: 85,
+        width,
+        transition: 'width 0.3s ease-in-out',
```

Comment:
Let's remove the `transition`. Since the width of a thought changes instantaneously while editing, we should update the divider width instantaneously so it matches.